src/components/ActionsPanel/components/CollapseActions.scss

```diff
@@ -0,0 +1,38 @@
+@use '../../variables';
+
+.#{variables.$ns}actions-panel-collapse {
+    $minSize: 32px;
+
+    flex-shrink: 2;
+    min-width: $minSize;
+    overflow: hidden;
+    position: relative;
+    display: flex;
+    align-items: center;
+    margin-right: 4px;
+
+    &__container {
+        display: flex;
+        align-items: center;
+        overflow: hidden;
```

Comment:
this style affects focused button
![image](https://user-images.githubusercontent.com/26501073/233352863-e2ad5da2-4659-4782-826a-7d33cc07ffca.png)

Reply:
fixed